components/Dialogs/src/private/MDCAlertControllerView+Private.m

```diff
@@ -353,8 +355,24 @@ - (CGSize)calculateActionsSizeThatFitsWidth:(CGFloat)boundingWidth {
 - (void)layoutSubviews {
   [super layoutSubviews];
 
-  for (UIButton *button in self.actionButtons) {
+  for (MDCFlatButton *button in self.actionButtons) {
     [button sizeToFit];
+    CGRect buttonFrame = button.frame;
+    buttonFrame.size.width =
+        MAX(CGRectGetWidth(buttonFrame), MDCDialogActionButtonMinimumWidth);
+    buttonFrame.size.height =
+        MAX(CGRectGetHeight(buttonFrame), MDCDialogActionButtonHeight);
+    button.frame = buttonFrame;
+    CGFloat verticalInsets =
+        -(MDCDialogActionButtonTouchTarget - CGRectGetHeight(button.frame)) / 2;
```

Comment:
super-nit: Can this be simplified to here and below?
(CGRectGetHeight(button.frame) - MDCTouchTarget) / 2